src/store/active.js

```diff
@@ -10,6 +11,7 @@ import {
 const state = () => ({
   type: null,
   id: null,
+  status: 'stopped', // can be 'playing' or 'paused' as well
```

Comment:
Just thinking out loud, but the distinction between `stop` and `pause` is maybe too subtle for this situation (they're synonyms outside the world of physical audio/video players)... Would `ejected` be clearer than `stopped`?

Reply:
I think finished/completed/ended could be good terms here. `ejected` is fun if I picture cassette tapes

Reply:
Most music players have a very explicit "stop" functionality (indicated by the square icon)

Reply:
I think finished/completed/ended would not be correct, because as per my usage here, we reach 'stopped' when the close button is pressed and the player disappears. In that sense, ejected makes more sense to me (and is fun to picture so win-win)

Reply:
Updated the name in 00243ec.